guardian/prism
# feat(apps-with-cdk-version): add field for which cdk pattern the app is using (if any)

## What does this change?


Work has recently been done to [add a tag to ASGs (and underlying instances) of which @guardian/cdk pattern is being used to generate it, if any](https://github.com/guardian/cdk/pull/603), allowing us to track which applications have been migrated to our patterns more easily. In order for this new data to be useful, we need to have it surface somewhere first. 

Since we're already tracking other CDK information on the `/apps-with-cdk-version` route, it makes the most sense to add this tag here. This PR adds the `gu:cdk:pattern-name` tag to the aforementioned route (as `guCdkPatternName`) so that we can track pattern usage in our [tracker lambda](https://github.com/guardian/tracker/blob/main/cdk/lib/guardian-cdk.ts).

## How to test


This has been deployed to CODE, so you can navigate to `<PRISM_URL>/apps-with-cdk-version?guCdkPatternName=guPlayApp` and see results!

## How can we measure success?


We can easily surface which GuCDK patterns are being used across our AWS estate.

## Have we considered potential risks?


Can't think of any.
